### DISCUSSION TITLE
Required codeowner reviews not always showing the `Reviewers` list

### DISCUSSION THREAD

**Discussion by nickofthyme** [community/community#118340] [Pull Requests]
### Select Topic Area

Bug

### Body

The other day I went to merge a PR and found that I could not merge it. So I go to the reviewers list and the top of the PR to see who's review was still needed, just to see the existing reviewers and a generic note saying `At least 1 approving review is required to merge this pull request`.

<img width="325" alt="image" src="https://github.com/community/community/assets/19007109/44937e77-9091-4665-b519-bbe5a7456344">

Then going to the bottom merge section of the PR, I see that there is still a missing review for a codeowner.

<img width="916" alt="image" src="https://github.com/community/community/assets/19007109/9534211d-57f4-4315-8a17-b5296116c001">

What changed here? Was this intentional? If so can we put this back?

**Comment by MarouaneBenbetka** [comment]
Yes this was intentional , the code owner added a protection rule that he needs to approve the pullrequest before it's merged , 
you can check the branch protection rules to disable or enable this option.

**Reply by nickofthyme** [reply]
I think you missed the point of my question, I understand the branch protection option to enforce codeowner review before merging.

<img width="575" alt="image" src="https://github.com/community/community/assets/19007109/08b67de1-dc7a-47a6-aa0a-5c3b4e416859">

The issue is that the `Reviewers` list used to always show the missing codeowner teams still needing to review the PR. So in the case I showed above the `Reviewers` list should look like this...


<img width="400" alt="image" src="https://github.com/community/community/assets/19007109/bfe9e10e-c700-4571-9ac5-48b2e5d105af">

The indication of the missing team review is gone in 2 PRs that I've seen in the past 2 weeks.
